rds/rds.tf

```diff
@@ -15,9 +16,11 @@ resource "aws_db_instance" "main" {
   multi_az                            = var.multi_az
   publicly_accessible                 = var.publicly_accessible
   deletion_protection                 = true
+
   vpc_security_group_ids = [
     aws_security_group.db.id,
   ]
+
```

Comment:
Why the extra spaces here? This looks really odd as there's no logical spearation of the attributes.

It makes sense for `tags` since we have them at the bottom of every resources, but thse are just generic key/value pairs.

Reply:
removed.